lib/twitter_cldr/js/renderers/numbers/rbnf.rb

```diff
@@ -0,0 +1,55 @@
+# encoding: UTF-8
+
+# Copyright 2012 Twitter, Inc
+# http://www.apache.org/licenses/LICENSE-2.0
+
+module TwitterCldr
+  module Js
+    module Renderers
+      module Numbers
+        module RBNF
+
+        	class RBNFRenderer < TwitterCldr::Js::Renderers::Base
```

Comment:
This class is called `RBNFRenderer` because it is the renderer for `rbnf.coffee` which is like the main class for RBNF. This file is called `rbnf.rb` because it has all the one liner renderers for other RBNF related modules. I bundled them all in here following your previous suggestion. Should I move them somewhere else or rename the file?